src/main/java/bbd/jportal2/generators/FreeMarker/FreeMarker.java

```diff
@@ -99,9 +99,10 @@ private Configuration configure(File templateDir) throws IOException {
         //templates (inside of src/main/resources/si_processor_templates/ and src/main/resources/postprocessor_templates/
         //as well as the location specified by the user
         FileTemplateLoader ftl1 = new FileTemplateLoader(templateDir);
+        FileTemplateLoader ftl2 = new FileTemplateLoader(new File(Paths.get("").toAbsolutePath().toString()));
```

Comment:
Do we need this? Surely it can just be specified by templateDir?

Reply:
The templateDir is based on the "root" of the selected template. 

E.g. specifying c:\templates as template-location and the template-generator as my_template, the templateDir at this point is: c:\templates\my_template . So if you put "common" ftl files in here, it'll form part of the output, essentially copying common template/helper files to the output.

Hence the need to "jump out".